packages/reporter/src/reporters/gitlab/api/GitLabReportSender.ts

```diff
@@ -0,0 +1,7 @@
+import type { CodeQualityReport } from '../types';
+
+export interface GitLabReportSender {
```

Comment:
name does not convey purpose and essence of this interface
```suggestion
export interface GitLabCIArtifacts {
```